addons/website_event_track_quiz/security/ir.model.access.csv

```diff
@@ -0,0 +1,7 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_event_quiz_all,event.quiz,model_event_quiz,,1,0,0,0
+access_event_quiz_event_manager,event.quiz manager,model_event_quiz,event.group_event_manager,1,1,1,1
+access_event_quiz_question_all,event_quiz_question,model_event_quiz_question,,1,0,0,0
```

Comment:
If everything goes through controllers, why not removing public access to quiz-related models ? If people can access track -> they gain access to the quiz info provided it has started (for non event people at least).